### PR TITLE
RIA-7524 - Merge master to naba branch

### PR DESCRIPTION
**Pull request by colmlynch60**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-7254

### Change description ###

Merged master changes to NABA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java [hunk 1]
```diff
@@ -1971,30 +2131,42 @@ public PreSubmitCallbackHandler<AsylumCase> ftpaApplicationDecisionRefusedOrNotA
...
-                            decision -> decision.toString().equals(FTPA_NOT_ADMITTED.toString())
-                                        || decision.toString().equals(FTPA_REFUSED.toString())
-                                        || decision.toString().equals(FTPA_REMADE32.toString()))
-                        .orElse(false);
-                }
 
                 return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
                        && (callback.getEvent() == Event.LEADERSHIP_JUDGE_FTPA_DECISION
                            || callback.getEvent() == Event.RESIDENT_JUDGE_FTPA_DECISION)
-                       && !AsylumCaseUtils.isInternalCase(asylumCase)
```
Comment (neha-aggarw) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java:1994]: This was missed here. We need this internal Check.

**Review by neha-aggarw — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java (hunk 1 above)
Comment (colmlynch60) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java:1994]: good spot. Added back

**Review by neha-aggarw — APPROVED** [review]
All good :)

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Pull request merged**